packages/tools/fluid-runner/src/fakeUrlResolver.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRequest } from "@fluidframework/core-interfaces";
+import { IContainerPackageInfo, IResolvedUrl, IUrlResolver } from "@fluidframework/driver-definitions";
+import { IOdspResolvedUrl } from "@fluidframework/odsp-driver-definitions";
+
+const fakeId = "b9d30c31-dd2a-4658-bd01-cf77a7dec25c";
```

Comment:
maybe use human readable string instead? Like fakeResolver? It's a bit easier for humans to realize that they can search for it in code base when they see it :)